file-formats/cota/cota/type/zif_aff_cota_v1.intf.abap

```diff
@@ -0,0 +1,92 @@
+INTERFACE zif_aff_cota_v1
+  PUBLIC.
+
+
+  TYPES:
+    "! <p class="shorttext">Configuration</p>
+    "! Configuration
+    "! $required
+    BEGIN OF ty_configuration,
+      "! <p class="shorttext">Connection Target Name</p>
+      "! Connection target name
+      "! $required
+      conn_id                     TYPE c LENGTH 30,
+      "! <p class="shorttext">Connection Target Class</p>
+      "! Connection target class
+      "! $required
+      connection_target_class     TYPE c LENGTH 30,
+      "! <p class="shorttext">Multiple Application Destinations Allowed</p>
+      "! Multiple application destinations allowed
+      allow_multiple_destinations TYPE abap_bool,
+      "! <p class="shorttext">Client Dependent Object</p>
+      "! Client dependent object
+      "! "! $required
+      client_dependant            TYPE abap_bool,
+    END OF ty_configuration.
+
+
+  "! <p class="shorttext">RFY Type</p>
+  "! $values {@link zif_aff_cota_v1.data:co_rfctype}
+  TYPES ty_rfctype TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">RFC Type</p>
+    "! RFC Type
+    BEGIN OF co_rfctype,
+      "! <p class="shorttext">Synchronous</p>
+      "! Synchronous
+      synchronous  TYPE ty_rfctype VALUE 'X',
+      "! <p class="shorttext">Asynchronous</p>
+      "! Asynchronous
+      asynchronous TYPE ty_rfctype VALUE 'S',
+    END OF co_rfctype.
+
+
+
+  TYPES:
+    "! <p class="shorttext">RFC Settings</p>
+    "! RFC Settings
+    "! $required
+    BEGIN OF ty_rfc_settings,
+      "! <p class="shorttext">Enforce Classic RFC</p>
+      "! Enforce classic RFC
+      "! $required
+      enforce_classic_rfc          TYPE abap_bool,
```

Comment:
Boolean fields are usually not required, because they are either true or false.

Same for `enforce_fast_serialization` and `usage_of_default_compression`